lib/TransportRPC.js

```diff
@@ -15,6 +15,13 @@ class TransportRPC extends TransportSocket {
   format (data) {
     return JSON.stringify(data)
   }
+
+  getSocket (port, conf) {
+    // https://github.com/uWebSockets/bindings/issues/23
+    throw new Error(
+      'implementation error: getSocket method needs to get extended'
+    )
+  }
```

Comment:
@prdn i added the `getSocket` method to `TransportRPC.js` to have it available in the inheriting `TransportRPCServer.js` classes